examples/backend-service/main.tf

```diff
@@ -94,6 +94,31 @@ module "ecs_service_definition" {
     registry_arn = aws_service_discovery_service.this.arn
   }
 
+  service_connect_configuration = {
+    enabled = "true"
+    namespace = data.aws_service_discovery_dns_namespace.this.id
```

Comment:
I just pushed a fix for this https://github.com/clowdhaus/terraform-aws-ecs/commit/e2d3a92a852dbda3cf935fea9db3a2711578860d

If you `terraform init -upgrade=true` to pull in the latest changes, it should resolve the error

Reply:
awesome, thanks Bryant, I'll test ASAP ^^